packages/shopping-cart/src/types.ts

```diff
@@ -681,6 +681,7 @@ export interface RequestCartProductExtra extends ResponseCartProductExtra {
 	signup?: boolean;
 	headstart_theme?: string;
 	feature_slug?: string;
+	is_hundred_year_domain?: boolean;
```

Comment:
Out of curiosity, why do we need to mark a product as being a hundred year domain? Shouldn't the `product_slug` and/or the `volume` be enough?

Reply:
Thanks for taking a look!

At least in this first iteration, we didn't want to create a new product for the sake of simplicity. We are still debating how we'll manage these 100-year domains, so for now I thought adding a flag to the product would be simpler. In the backend side, we'll mark these domains with a flag too.

Using `volume` might work, but I'm not sure how that would interact with the variations and pricing 🤔 I'd need to investigate some more.

Reply:
Ok, it's not a big problem but I was curious. If it's re-using the same product ID as other domains, we might be able to use `volume` since that's how multi-year domain purchases work normally, even if this might be presented differently. Anyway, I'm happy to help. If the pricing is different for a 100 year domain than it would be if you purchased 100 years of the same domain, then how that pricing is applied (different billing plan, discount, or cost override) will make all the difference.